tools/sdk-tools/adpcm/vpredictor.c

```diff
@@ -9,8 +9,8 @@ s32 readcodebook(FILE *fhandle, s32 ****table, s32 *order, s32 *npredictors)
     s32 j;
     s32 k;
 
-    fscanf(fhandle, "%d", order);
-    fscanf(fhandle, "%d", npredictors);
+    if (fscanf(fhandle, "%d", order)){};
```

Comment:
Same old unused result shit.

Reply:
again, checking for the return value here and failing if incorrect would solve this once and for all

Reply:
Yet again resolved by using these as error-checkers.